examples/live/check_results.py

```diff
@@ -1,208 +1,236 @@
+                end_time = group['end_time'][:]
+                if (end_time < sim_gps_start).any() or (end_time > sim_gps_end).any():
+                    log.error('Invalid merger time in %s %s', trigfp, detector)
+                    single_fail = True
+
+                # check that template parameters are consistent with the bank
+                trig_mass1 = group['mass1'][:]
+                trig_mass2 = group['mass2'][:]
+                trig_s1z = group['spin1z'][:]
+                trig_s2z = group['spin2z'][:]
+                trig_temp_id = group['template_id'][:]
+                test_mass1 = close(temp_mass1[trig_temp_id], trig_mass1, 1e-7)
+                test_mass2 = close(temp_mass2[trig_temp_id], trig_mass2, 1e-7)
+                test_s1z = close(temp_s1z[trig_temp_id], trig_s1z, 1e-7)
+                test_s2z = close(temp_s2z[trig_temp_id], trig_s2z, 1e-7)
+                test_all = np.logical_and.reduce((test_mass1, test_mass2,
                                               test_s1z, test_s2z))
-                        if not test_all.all():
-                              log.error('Invalid template parameters in %s %s',
-                                    trigfp, detector)
-                              single_fail = True
-
-      # check that triggers were produced in all detectors
-      if detectors_with_trigs != tested_detectors:
-            missing = sorted(tested_detectors - detectors_with_trigs)
-            log.error('No triggers found in %s', ', '.join(missing))
-            single_fail = True
+                if not test_all.all():
+                    log.error('Invalid template parameters in %s %s',
+                              trigfp, detector)
+                    single_fail = True
+
+    # check that triggers were produced in all detectors
+    if detectors_with_trigs != tested_detectors:
+        missing = sorted(tested_detectors - detectors_with_trigs)
+        log.error('No triggers found in %s', ', '.join(missing))
+        single_fail = True
       
-      if single_fail:
-            log.error('Single Trigger Test Failed')
-      return single_fail
+    if single_fail:
+        log.error('Single Trigger Test Failed')
+    return single_fail
+
 
 def check_coinc_results(inj_table):
-      coinc_fail = False
-      print_template_parameters = False
-      # gather coincident triggers
-      coinc_trig_paths = sorted(glob.glob('output/coinc*.xml.gz'))
-      l=len(coinc_trig_paths)
-      if l==0:
+    coinc_fail = False    
+    # gather coincident triggers
+    coinc_trig_paths = sorted(glob.glob('output/coinc*.xml.gz'))
+    n_coincs=len(coinc_trig_paths)
+    if n_coincs==0:
+        if verbose:
             log.error('No coincident triggers detected')
-            coinc_fail = True
-      elif l>=10: 
+        coinc_fail = True
+    elif n_coincs>=10: 
+        if verbose:
             log.error('Too many coincident triggers detected')
```

Comment:
Errors should be displayed regardless of any verbose flag.

Reply:
Based on your other comment, should this even produce an error at all? If we "run over a lot of data or a very large bank" we could get 10 triggers?